src/action/bandaidset.rs

```diff
@@ -47,15 +47,36 @@ impl TryFrom<(String, &Span)> for FirstAidKit {
         if span.is_multiline() {
             bail!("Can't construct `FirstAidKit` from single-line span only")
```

Comment:
`bail!` text is in conflict with condition check

Reply:
good catch, thanks!